unitTests/utils/DomTest.ts

```diff
@@ -652,7 +781,8 @@ export function DomTests() {
       it('using "on" with non alpha numeric character should work properly', () => {
         const spy = jasmine.createSpy('spy');
         new Dom(el).on('this contains space', spy);
-        window['Coveo']['$'](el).trigger('thiscontainsspace');
+        let event = new CustomEvent('thiscontainsspace');
```

Comment:
Can be `const`.

I am also not sure why this test needed to be changed ? 
It seems this test was specifically testing for jQuery events, and the nature of the test is completely different now.

Shouldn't it instead be a new suite of test if you want to test that ?

Reply:
Not as stated in this comment https://github.com/coveo/search-ui/pull/929/files#diff-5613055c90d6a93514af0f50d75b89ccR642.

Should i delete the comment and go back to JQuery event ?